mappings/net/minecraft/test/TestContext.mapping

```diff
@@ -282,6 +284,7 @@ CLASS net/minecraft/class_4516 net/minecraft/test/TestContext
 		ARG 2 pos
 	METHOD method_36039 toggleLever (Lnet/minecraft/class_2338;)V
 		ARG 1 pos
+	METHOD method_36040 forEachTickLeft (Ljava/lang/Runnable;)V
```

Comment:
This should be in line with Java naming conventions (`Iterator#forEachRemaining`):

```suggestion
	METHOD method_36040 forEachRemainingTick (Ljava/lang/Runnable;)V
```